components/automate-chef-io/content/docs/restore.md

```diff
@@ -0,0 +1,120 @@
++++
+title = "Restore"
+description = "Procedures for Restoring Chef Automate Data"
+date = 2018-03-26T15:27:52-07:00
+draft = false
+bref = ""
+toc = true
+[menu]
+  [menu.docs]
+    parent = "get_started"
+    weight = 80
++++
+
+Restore Chef Automate from a [filesystem backup]({{< ref "restore.md#restore-from-a-filesystem-backup" >}}) or an [Amazon S3 bucket backup]({{< ref "restore.md#restore-from-an-s3-backup" >}}).
+
+## Prerequisites
+
+1. On the restore host, download and unzip the Chef Automate command-line tool:
+
+   ```shell
+        curl https://packages.chef.io/files/current/latest/chef-automate-cli/chef-automate_linux_amd64.zip | gunzip - > chef-automate && chmod +x chef-automate
+    ```
+
+1. **Filesystem backups** require access for Chef Automate to a backup directory in the [configured location]({{< ref
+"backup.md#backup-to-a-filesystem" >}}).
+Ensure access for the backup type used:
+
+     1. To restore a network-attached filesystem backup, mount the shared backup directory to the same mount point configured at the time of the backup.
```

Comment:
@yzl Does Line 28 not include a link to "backing up a file system" like lines 29 and 30 because the same mount point would be used regardless?

Reply:
It doesn't hurt to add the link, but yes.

Reply:
@yzl This link is already repeated in the opening paragraph on line 24. (I somehow broke that link in editing, but I'll fix that!) I'm going to keep it simple by removing the links in line 29 and 30 and keeping the link in line 24.

If anything, I would consider instead adding links for each backup type and their relevant restore section.

Reply:
@yzl So would the following mapping be correct? I'm putting in brackets what the hyperlinked text would be for lines 28-31.
To  restore a [network-attached file system backup]: docs/backup/#backup-to-s3
To restore a [backup directory that is not a network-attached filesystem]: docs/backup/#backup-to-a-filesystem
For [restoring a single-file backup archive] : /docs/backup/#store-a-filesystem-backup-in-a-single-file-archive

Reply:
@mjingle Not quite.

Automate does not distinguish between backing up to a local filesystem and backing up to a network-attached filesystem. If you wanted to add links, the "network-attached filesystem backup" case and the "backup directory that is not a network-attached filesystem" (you could say "backup directory on a local filesystem") are both filesystem backups, so they should both point to docs/backup/#backup-to-a-filesystem.

What you have for the single-file backup archive is fine.

Reply:
Now I'm following why you included the links that you did. Thank you for your patience as I got to that point. I'll amend accordingly.